frontend/package.json

```diff
@@ -5,7 +5,6 @@
   "dependencies": {
     "@testing-library/jest-dom": "^4.0.0",
     "@testing-library/react": "^9.1.0",
-    "jest-dom": "^4.0.0",
```

Comment:
Why don't we need it anymore? 🤔

Reply:
It's deprecated, the current version is on line 6 of package.json